testcases/OpTestEEH.py

```diff
@@ -207,12 +224,26 @@ def check_eeh_slot_resets(self):
     # @returns True/False @type boolean
     #
     def check_eeh_pe_recovery(self, pe):
-        list = self.get_list_of_pci_devices()
-        for device in list:
-            if pe in device:
-                return True
-        return False
-
+        cmd = "dmesg  | grep -i 'EEH: Notify device driver to resume'; echo $?"
+        tries = 60
+        for i in range(1, tries+1):
+            res = self.cv_SYSTEM.sys_get_ipmi_console().run_command(cmd)
+            if int(res[-1]):
+                print "Waiting for PE %s EEH Completion: (%d/%d)" % (pe, i, tries)
+                time.sleep(1)
+            else:
+                break
+        else:
+            raise EEHRecoveryFailed("EEH recovery failed") 
```

Comment:
here it is failing with below error.

======================================================================
ERROR: runTest (testcases.OpTestEEH.OpTestEEHbasic_frozen_pe)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/pridhiviraj/new_optests/venkat_eeh_patch/op-test-framework/testcases/OpTestEEH.py", line 425, in runTest
    elif not self.check_eeh_pe_recovery(pe):
  File "/home/pridhiviraj/new_optests/venkat_eeh_patch/op-test-framework/testcases/OpTestEEH.py", line 237, in check_eeh_pe_recovery
    raise EEHRecoveryFailed("EEH recovery failed")
TypeError: __init__() takes at least 3 arguments (2 given)

Change the statement like this
raise EEHRecoveryFailed("EEH recovery failed", pe)